jmeter/plans/apply.rb

```diff
@@ -0,0 +1,92 @@
+require 'ruby-jmeter'
+
+BASEURL = ENV.fetch('JMETER_TARGET_BASEURL')
+
+def url(path)
+  BASEURL + path
+end
+
+# 975 concurrent users, ~7 minutes per session
```

Comment:
Picked a tidy number close to the projected concurrent user count, this could potentially be upped to 1000.